lang/java/archetypes/avro-service-archetype/src/main/pom/pom.xml

```diff
@@ -33,6 +33,8 @@
   <name>Simple Avro Ordering Service</name>
 
   <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Should the values be references to the ones in the main/parent pom (https://github.com/apache/avro/blob/fcc4e2dec64e8cbb011a388f5e612c2b09e4d3ef/pom.xml#L43-L44) ?
Either `${...}` or `@...@` (see https://github.com/apache/wicket/blob/master/archetypes/quickstart/src/main/resources/archetype-resources/pom.xml#L46)